### DISCUSSION TITLE
[Question] How i make a nested query?

### DISCUSSION THREAD

**Discussion by approached** [wovalle/fireorm#263] [Q&A] [answered]
Hi@all 

I have a document/company with id 'Fire LLC'.  But how i can query my members collection? I don't want first query my company collection and than the members collection. I have already my company id.

My Example:

```js
    const result = admin
        .firestore()
        .collection('companies')
        .doc('Fire LLC')
        .collection('members)
        .get()
```

**Comment by wovalle** [comment, marked as answer]
Hey! As of today, there is no way of preloading subcollections, so you'll have fetch the collection first.

```js

@Collection('companies')
class Company {}

const companyRepository = getRepository(Company);

const company = companyRepository.getById('Fire LLC');
const members = company.members.find()

```
